src/sfdt/toggleBookmark.ts

```diff
@@ -5,6 +5,88 @@ import {
 	isBookmarkStart,
 	isBookmarkEnd,
 } from '../queryBookmark'
+import Stack from '../stack'
+
+const makeToggleOff = (inlines, name) => {
+  const newInlines = []; // Should act as queue for getting new list of inlines
+  const stackOfBookmarks = new Stack(); // will be used to find relationship of bookmarks
+  let toggleOffAtIndex = null;
+
+  inlines.forEach((inline, index) => {
+    const nextInline = inlines[index + 1];
+    const prevInline = inlines[index - 1];
+
+    if (!(inline.fieldType || inline.hasFieldEnd)) {
+      newInlines.push(inline);
+    }
+
+    if (inline.name && inline.name.split("::").includes("COND")) {
```

Comment:
this function should not know what a 'COND' is. please refactor so we make this a generic toggleBookmark function, not hardcoded to our usecase.